teacher.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'person'
+class Teacher < Person
+  attr_reader :salary, :target_raise, :raise_performance_threshold
+
+  def initialize(options={})
+    super
+    @phase = 3
+    @raise_performance_threshold = 90
+  end
+
+  def receive_raise(raise)
+    @salary += raise
+  end
+
+  def set_performance_rating(rating)
```

Comment:
Setting `response` as a local variable then appending to it looks a little odd. Probably OK to just return the proper string at the time a particular condition is hit.